fetch.bs

```diff
@@ -231,6 +231,9 @@ lt="authentication entry">authentication entries</a> (for HTTP authentication).
 
  <dt><dfn for="fetch controller">report timing steps</dfn> (default null)
  <dd>Null or an algorithm accepting a <a for=/>global object</a>.
+
+ <dt><dfn for="fetch controller">cloned error</dfn> (default "<code><a exception>AbortError</a></code>" {{DOMException}})
```

Comment:
Maybe `abort reason`, and null as a default?

Reply:
Done. I made the default `null`.